*** menu.cfg	2015-07-22 15:01:12.752856243 -0700
--- menu.cfg	2015-07-22 16:07:59.063129882 -0700
***************
*** 1,17 ****
! menu hshift 0
! menu width 82
  
  menu title Boot menu
  include stdmenu.cfg
  include live.cfg
- include install.cfg
- menu begin advanced
- 	menu title Advanced options
- 	include stdmenu.cfg
- 	label mainmenu
- 		menu label ^Back..
- 		menu exit
- 	include advanced.cfg
- menu end
  
  menu clear
--- 1,8 ----
! menu hshift 8 
! menu width 45
  
  menu title Boot menu
  include stdmenu.cfg
  include live.cfg
  
  menu clear
*** stdmenu.cfg	2015-07-22 15:06:35.295521054 -0700
--- stdmenu.cfg	2015-07-22 16:07:26.363666085 -0700
***************
*** 5,11 ****
  menu color hotsel	1;7;37;40 #ffffffff #76a1d0ff *
  menu color tabmsg	* #ffffffff #00000000 *
  menu color help		37;40 #ffdddd00 #00000000 none
! menu vshift 12
  menu rows 10
  menu helpmsgrow 15
  # The command line must be at least one line from the bottom.
--- 5,11 ----
  menu color hotsel	1;7;37;40 #ffffffff #76a1d0ff *
  menu color tabmsg	* #ffffffff #00000000 *
  menu color help		37;40 #ffdddd00 #00000000 none
! menu vshift 16
  menu rows 10
  menu helpmsgrow 15
  # The command line must be at least one line from the bottom.
*** isolinux.cfg	2015-07-22 15:07:42.506409383 -0700
--- isolinux.cfg	2015-07-22 15:07:57.566160125 -0700
***************
*** 1,4 ****
  include menu.cfg
  default vesamenu.c32
  prompt 0
! timeout 0
--- 1,4 ----
  include menu.cfg
  default vesamenu.c32
  prompt 0
! timeout 100
